markets/storageadapter/dealpublisher.go

```diff
@@ -315,6 +318,13 @@ func (p *DealPublisher) publishReady(ready []*pendingDeal) {
 // validateDeal checks that the deal proposal start epoch hasn't already
```

Comment:
update comment (it's doing more now)

Reply:
@magik6k can we add tests like @arajasek suggested before we take things further?

Reply:
FYI - @BigLep

Reply:
Thanks for adding the test and holding the standard for ourselves and others.